quickstart.md

```diff
@@ -270,12 +270,284 @@ In case the user wants to run the workload on a TDX capable hardware, using QEMU
 
 ### SEV
 
-`kata-qemu-sev`
-Export cert chain
-Start KBS (even for unencrypted)
+#### Install sevctl and Export SEV Certificate Chain
 
-Use the sample image or an unencrypted image
+[sevctl](https://github.com/enarx/sevctl) is the SEV command line utility and is needed to export the SEV certificate chain.
 
+Rust must be installed to build this tool:
```

Comment:
It might be worth noting that the tool can be installed on Fedora 36 or RHEL 9 with `dnf install sevctl`. Not sure what other distros have it.

Reply:
Fixed this. Done.